dll/shellext/sendmail/sendmail.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * PROJECT:     sendmail
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     DeskLink implementation
+ * COPYRIGHT:   Copyright 2019 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "precomp.hpp"
+
+WINE_DEFAULT_DEBUG_CHANNEL(sendmail);
+
+BEGIN_OBJECT_MAP(ObjectMap)
+    OBJECT_ENTRY(CLSID_DeskLinkDropHandler, CDeskLinkDropHandler)
+END_OBJECT_MAP()
+
+CComModule gModule;
+
+STDAPI DllCanUnloadNow(void)
```

Comment:
If this function returns `S_OK` it means the dll can be safely unloaded.
As long as there are `CDeskLinkDropHandler` active, this should not return `S_OK`.
See for example fontext or zipfldr that use a global `g_ModuleRefCnt` that is incremented in the constructor of object, end decremented in the destructor.
Then, when this value is not null, `S_FALSE` is returned in `DllCanUnloadNow`

Reply:
OK.